firefox-ios/Client/Frontend/Browser/Tabs/State/TabTrayState.swift

```diff
@@ -104,6 +104,14 @@ struct TabTrayState: ScreenState, Equatable {
                                 normalTabsCount: state.normalTabsCount,
                                 hasSyncableAccount: state.hasSyncableAccount,
                                 shareURL: shareURL)
+        case TabPanelAction.refreshTab(let tabModel, _):
+            // Only update the nomal tab count if the tabs being refreshed are not private
+            let isPrivate = tabModel.first?.isPrivate ?? false
+            let tabCount = isPrivate ? state.normalTabsCount : String(tabModel.count)
```

Comment:
Does this string created here need to use the same logic as `TabManagerMiddleware.normalTabsCountText`? It looks like we're using a String for the count instead of an Int because we swap in a special character once it goes into triple digits but I'm not entirely familiar with that code or whether it should apply to this `normalTabsCount` string.

Reply:
Good catch, yes it should use the same logic. I will update.